apps/style/common.scss

```diff
@@ -266,22 +267,30 @@ button.arrow:disabled {
     padding-inline-end: 9px;
   }
 }
+#soft-buttons.hidden {
```

Comment:
we should take advantage of SCSS' nesting functionality here for readability:

```scss
#soft-buttons {
  &.hidden {
    display: none;
  }
}
```

Reply:
Done.